packages/front-end/components/Experiment/ResultMoreMenu.tsx

```diff
@@ -151,6 +160,9 @@ export default function ResultMoreMenu({
           <FaPencilAlt className="mr-2" /> Add/Remove Metrics
         </button>
       )}
+      {results && (
```

Comment:
I don't think we need to limit this based on a user's permissions, but I can do so easily if anyone disagrees. I think the only user type I could see an argument for excluding is the `read-only` user.